packages/app/src/runner/dom.cy.tsx

```diff
@@ -0,0 +1,58 @@
+import { getOrCreateHelperDom } from './dom'
```

Comment:
This function seemed to be completely untested, so I added a test for this. Many of these runner 'helper' files are untested actually so not a lot of conventions to pull from.